js/source/raster_tile_source.js

```diff
@@ -51,10 +51,13 @@ RasterTileSource.prototype = util.inherit(Evented, {
         function done(err, img) {
             delete tile.request;
 
-            if (tile.aborted)
-                return;
+            if (tile.aborted) {
+                this.state = 'unloaded';
+                return callback(null);
+            }
```

Comment:
Is this fixing another bug? If so, thank you! Out of curiosity: what were the symptoms of this bug? I'm guessing something along the lines of "`map.loaded()` doesn't return true if there are any tiles were aborted."